projects/1.16.1/assets/atmospheric/atmospheric/lang/zh_cn.json

```
@@ -0,0 +1,430 @@
+	"item.minecraft.tipped_arrow.effect.worsening_long": "恶化之箭",
+	
+	"biome.atmospheric.rainforest": "雨林",
+	"biome.atmospheric.rainforest_mountains": "雨林山地",
+	"biome.atmospheric.rainforest_plateau": "雨林高原",
+	"biome.atmospheric.sparse_rainforest_plateau": "稀疏雨林高原",
+	"biome.atmospheric.rainforest_basin": "雨林盆地",
+	"biome.atmospheric.sparse_rainforest_basin": "稀疏雨林盆地",
+	
+	"entity.atmospheric.passionfruit_seed": "百香果种子",
+	
+	"death.attack.passionfruitSeed": "%1$s被%2$s吐出的百香果种子雨射死了",
+	"death.attack.shimmeringPassionfruitSeed": "%1$s被%2$s疯狂吐出的百香果种子雨射死了",
+	
+	"advancements.husbandry.spit_passionfruit.title": "噗——",
+	"advancements.husbandry.spit_passionfruit.description": "用吃百香果吐出的百香果种子击中敌人",
+
+  "_comment2": "START OF THE SPRINGS SECTION",
+
+	"block.atmospheric.ivory_travertine": "象牙白石灰华",
+	"block.atmospheric.peach_travertine": "桃色石灰华",	
+	"block.atmospheric.persimmon_travertine": "柿子色石灰华",
+	"block.atmospheric.saffron_travertine": "藏红石灰华",	
+	
+	"block.atmospheric.chiseled_ivory_travertine": "錾制象牙白石灰华",
+	"block.atmospheric.chiseled_peach_travertine": "錾制桃色石灰华",	
+	"block.atmospheric.chiseled_persimmon_travertine": "錾制柿子色石灰华",
+	"block.atmospheric.chiseled_saffron_travertine": "錾制藏红石灰华",	
+	
+	"block.atmospheric.cut_ivory_travertine": "切制象牙白石灰华",
+	"block.atmospheric.cut_peach_travertine": "切制桃色石灰华",	
+	"block.atmospheric.cut_persimmon_travertine": "切制柿子色石灰华",
+	"block.atmospheric.cut_saffron_travertine": "切制藏红石灰华",	
+	
+	"block.atmospheric.ivory_travertine_stairs": "象牙白石灰华楼梯",
+	"block.atmospheric.peach_travertine_stairs": "桃色石灰华楼梯",	
+	"block.atmospheric.persimmon_travertine_stairs": "柿子色石灰华楼梯",
+	"block.atmospheric.saffron_travertine_stairs": "藏红石灰华楼梯",	
+	
+	"block.atmospheric.ivory_travertine_slab": "象牙白石灰华台阶",
+	"block.atmospheric.peach_travertine_slab": "桃色石灰华台阶",	
+	"block.atmospheric.persimmon_travertine_slab": "柿子色石灰华台阶",
+	"block.atmospheric.saffron_travertine_slab": "藏红石灰华台阶",	
+	
+	"block.atmospheric.ivory_travertine_wall": "象牙白石灰华墙",
+	"block.atmospheric.peach_travertine_wall": "桃色石灰华墙",	
+	"block.atmospheric.persimmon_travertine_wall": "柿子色石灰华墙",
+	"block.atmospheric.saffron_travertine_wall": "藏红石灰华墙",	
+	
+	"block.atmospheric.ivory_fumarole_block": "象牙白气孔方块",
+	"block.atmospheric.peach_fumarole_block": "桃色气孔方块",	
+	"block.atmospheric.persimmon_fumarole_block": "柿子色气孔方块",
+	"block.atmospheric.saffron_fumarole_block": "藏红气孔方块",	
+	
+	"block.atmospheric.ivory_travertine_vertical_slab": "象牙白石灰华竖直台阶",
+	"block.atmospheric.peach_travertine_vertical_slab": "桃色石灰华竖直台阶",	
+	"block.atmospheric.persimmon_travertine_vertical_slab": "柿子色石灰华竖直台阶",
+	"block.atmospheric.saffron_travertine_vertical_slab": "藏红石灰华竖直台阶",	
+	
+	"item.atmospheric.spring_water": "泉水",
+	"item.atmospheric.combustive_tonic": "易燃的奎宁水",
+	"item.atmospheric.charged_tonic": "带电的奎宁水",
+
+	"biome.atmospheric.hot_springs": "热泉",
+	
+	"death.attack.fumarole": "%1$s试图靠近喷泉但被轰开了",
+	"death.attack.fumarole.player": "%1$s在试图逃离%2$s时试图靠近喷泉",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

两个试图，不妥。

```suggestion
	"death.attack.fumarole.player": "%1$s在逃离%2$s时试图靠近喷泉",

```

---

已修正